src/networkDetails.ts

```diff
@@ -0,0 +1,407 @@
+import arbitrum from "./assets/arbitrum.png";
+import avalanche from "./assets/tokens/AVAX.svg";
+import polygon from "./assets/tokens/matic.svg";
+import ethereum from "./assets/tokens/wETH.svg";
+import { EnvHelper } from "./helpers/Environment";
+import { NodeHelper } from "./helpers/NodeHelper";
+
+export enum NetworkId {
```

Comment:
Can we keep this in constants.ts file?

Reply:
@mrunbanked `NetworkId` and `IAddresses` are imported and re-exported in `constants.ts`

Does that achieve what you're looking for?